src/contexts/toast/Toast.tsx

```diff
@@ -98,7 +99,9 @@ function Toast({ message, onClose, variant }: Props) {
             <AlertIcon />
           </StyledAlertIcon>
         )}
-        <BaseM>{message}</BaseM>
+        <BaseM>
+          <Markdown text={message} />
+        </BaseM>
```

Comment:
Added markdown support. Been thinking to support `NodeElement` but it seems lot of refactor needed

Reply:
nice one!!